### PR TITLE
fix: vertical paddings of buttons in settings

### LINKED ISSUE #519: Incorrectly vertically padded buttons in settings
image

The buttons have less space at the red lines than they do between each other or settings toggles have between themselves

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
Increases margin top and bottom

### Tests for unexpected behavior
**Before**
image

**After**
image

### Time spent on PR
5 mins

### Linked issues
Fix #519 

### Reviewers
@bhajneet @Harjot1Singh

### CODE REVIEW

**Comment by Harjot1Singh** [comment]
@bhajneet does this look ok to you?

**Comment by bhajneet** [comment]
@saihaj can you please share an image of the first two buttons being used vertically with this vertical-padding-fix?

**Comment by bhajneet** [comment]
Also why is there no single class for buttons of this type in settings? It should not require setting padding in two css selectors.

**Comment by saihaj** [comment]
> @saihaj can you please share an image of the first two buttons being used vertically with this vertical-padding-fix?

#586 will do that

File: app/frontend/src/Settings/OverlaySettings.css [hunk 1]
```diff
@@ -4,9 +4,9 @@
 }
 
 .overlay-settings .tutorial-button {
-  margin: 15px 0;
+  margin: 18px;
```
Comment (Harjot1Singh) [inline app/frontend/src/Settings/OverlaySettings.css:7]: Why did you take off the 0 horizontal padding?

**Review by Harjot1Singh — CHANGES_REQUESTED** [review]

**Review by Harjot1Singh — APPROVED** [review]

**Pull request merged**